introduction.md

```diff
@@ -0,0 +1 @@
+Hello, I am Moondoobie. I have a degree in Electrical Engineering and an MBA, and I really like learning. I live in the Northwest U.S..
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.